examples/stock/company.v.py

```diff
@@ -1,21 +1,23 @@
 # Own shares of a company!
 company: address
-avail_shares: num
-total_shares: num
-price: wei_value
+total_shares: currency_value
+price: num (wei / currency)
 
```

Comment:
Every EVM call, or every Viper call?

Interesting, so some of the overestimating viper has is for checks that don't require state modification and therefore do not cost gas, is that correct?